src/components/chosenCard/ChosenCountry/ChosenCountryDesktop/ChosenCountryDesktop.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { GreetingCountryDesktop } from './GreetingCountryDesktop/GreetingCountryDesktop';
+import './chosen-country-desk.scss';
+import { ChosenCountryImageDesktop } from './ChosenCountryImageDesktop/ChosenCountryImageDesktop';
+
+export const ChosenCountryDesktop = ({ data }) => (
+  <div className="chosen-country">
+    <GreetingCountryDesktop />
+    <div className="chosen-coutry__content">
+      <div className="chosen-country__name-block">
+        <div className="chosen-country__title">Country</div>
+        <div className="chosen-country__name">{data.country.name}</div>
+      </div>
+
+      <div className="chosen-country__center">
+        <div className="chosen-country__left">
+
+          <div className="chosen-country__capital-block">
```

Comment:
Create new component:

```suggestion
export const ChosenCountryLabel ({title, value, className}) => (
          <div className="chosen-country-label ${className}">
              <span>{title}</span>
              {value}
          </div>    
```
```suggestion
          <ChosenCardLabel className="chosen-country__capital-block" title={..} ={value} />
```